src/components/Configuration/components.js

```diff
@@ -28,17 +28,17 @@ const Card = ({ body }) => {
   );
 };
 Card.propTypes = {
-  body: PropTypes.node
+  body: PropTypes.node,
 };
 export class Details extends Component {
   static propTypes = {
     url: PropTypes.string,
-    children: PropTypes.arrayOf(PropTypes.node)
-  }
+    myChilds: PropTypes.arrayOf(PropTypes.node),
```

Comment:
Why this change?

Reply:
![image](https://user-images.githubusercontent.com/1091472/106746499-9a33a680-665d-11eb-8c3c-187ea5522e59.png)

Reply:
Is this a new warning?
Or you want to fix it in this PR?

Reply:
Not a new one, before there's a `// eslint-disable-next-line` to disable the warning, see https://github.com/webpack/webpack.js.org/blob/master/src/components/Configuration/Configuration.jsx#L11